# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: ec0e9a72aebb
# Full Hash: ec0e9a72aebb72422b06c1cadee2b3f7dc3538ce
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2020-01-10 05:03:08
# Regressor Bug: 1600472
# File Overlap Count: 2
# Description:
#   Bug 1600472 - Disable allowing sacrificing subpixel anti-aliasing for small screens. r=jrmuizel
#   
#   This patch only allows sacrificing subpixel anti-aliasing when the
#   screen size is larger than WUXGA, and when the force disable pref is not
#   set. In the future, we may also add disable this for high end GPUs.
# ==============================================================================

diff -r 912fed7c5710 -r ec0e9a72aebb modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Thu Jan 09 14:14:08 2020 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Thu Jan 09 15:23:38 2020 +0000
@@ -3814,6 +3814,13 @@
 #endif
   mirror: always
 
+  # Force disables allowing sacrificing subpixel anti-aliasing for less
+  # powerful configurations.
+- name: gfx.webrender.quality.force-disable-sacrificing-subpixel-aa
+  type: bool
+  value: false
+  mirror: always
+
 # Use vsync events generated by hardware
 - name: gfx.work-around-driver-bugs
   type: bool
